var/lib/delphix-platform/ansible/apply

```diff
@@ -106,7 +105,8 @@ if [[ "$(find_playbooks | wc -l)" -eq "0" ]]; then
 	exit 1
 fi
 
-find_playbooks | while read -r playbook; do
+playbooks=$(find_playbooks) || exit 1
+for playbook in $playbooks; do
```

Comment:
@basil's proposed solution looks a little cleaner to me:
```
while read -r playbook; do
        apply_playbook "$playbook"
done < <(find_playbooks)
```
since it avoids creating a new variable.

But I'm OK with this too; either way, it's not a big deal, as long as the script properly reports errors, now.

Reply:
Process substitution does look cleaner, but we still have `find_playbooks` running in a sub-shell, so we have to check that sub-shell's exit status. I just wasn't aware of a simple way to do that.